apps/k8s-io/test.py

```diff
@@ -212,6 +211,13 @@ def test_go(self):
             self.assert_temp_redirect(
                 base + 'needs-ok-to-test',
                 'https://github.com/pulls?q=org%3Akubernetes+org%3Akubernetes-sigs+org%3Akubernetes-csi+org%3Akubernetes-client+is%3Aopen+is%3Apr+label%3Aneeds-ok-to-test+label%3A%22cncf-cla%3A+yes%22+-label%3Aneeds-rebase')
+            self.assert_temp_redirect(
+                base + 'contact',
+                'https://github.com/kubernetes/community/tree/master/communication')
```

Comment:
```suggestion
                'https://github.com/kubernetes/community/tree/master/communication/')
```
nit: just so that it is consistent with what we have in our nginx config.